src/ZConfig/tests/support.py

```diff
@@ -79,12 +78,6 @@ def f2(self):
 class TestHelper:
     """Utility methods which can be used with the schema support."""
 
-    # Not derived from unittest.TestCase; some test runners seem to
-    # think that means this class contains tests.
```

Comment:
I am not exactly sure if this comment is related to removed `getattr`. I can return back if it should stay there.